sandbox/examples/extra/_grading/grade.rb

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+
+require 'steno'
+
+fs = ['Makefile', 'test.pl']
+
+FileUtils.cp("test.pl", "_grading")
```

Comment:
Should be something like 

    FileUtils.cp(ENV["XTR"], "_grading/test.pl")

To demonstrate proper parameterization. Check driver.pl.erb for details.